.template/addons/bootstrap/template.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+use_source_path __dir__
+
+apply '.template/addons/bootstrap/application.scss.rb'
+apply '.template/addons/bootstrap/.scss_lint.yml.rb'
+apply '.template/addons/bootstrap/package.json.rb'
+apply '.template/addons/bootstrap/application.js.rb'
+
+directory('stylesheets/bootstrap', 'vendor/assets/stylesheets/bootstrap')
```

Comment:
Could you please explain more about why you decide to create bootstrap stylesheet ourselves instead of install bootstrap.
Thank you.

Reply:
Changed the implementations in [da9cb2f](https://github.com/nimblehq/rails-templates/pull/239/commits/da9cb2fda5a12c9cf2c6879281d8bc3044832bbb) as per the [comment](https://github.com/nimblehq/rails-templates/pull/239#discussion_r502198123).

Let me know your thoughts.

cc. @andyduong1920

Reply:
I commented my thought [here](https://github.com/nimblehq/rails-templates/pull/239#discussion_r502236835)